app/bundles/PageBundle/Model/PageModel.php

```diff
@@ -801,8 +801,10 @@ public function getHitsLineChartData($unit, \DateTime $dateFrom, \DateTime $date
     }
 
     /**
+     * @deprecated method not usefully and not use anymore
```

Comment:
This may be more valuable message.
```suggestion
     * @deprecated Use getUniqueVsReturningPieChartData() instead.
```